watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1575_magprex/magprex.py

```diff
@@ -257,22 +257,6 @@ def add_costing(m):
     )
 
 
-def initialize_system(m):
-    seq = SequentialDecomposition()
-    seq.options.tear_set = []
-    seq.options.iterLim = 1
-    seq.run(m, lambda u: u.initialize())
-
-
-def solve(blk, solver=None, tee=False, check_termination=True):
-    if solver is None:
-        solver = get_solver()
-    results = solver.solve(blk, tee=tee)
-    if check_termination:
-        assert_optimal_termination(results)
-    return results
-
-
 def display_results(m):
```

Comment:
In addition to `initialize_system` and `solve`, it looks like there also is a duplicate instance of `display_results`, so I recommend removing this one.